subprojects/analysis-kotlin-descriptors/compiler/src/main/kotlin/org/jetbrains/dokka/analysis/kotlin/descriptors/compiler/configuration/resolve/DokkaJsResolverForModuleFactory.kt

```diff
@@ -61,13 +63,14 @@ internal class DokkaJsResolverForModuleFactory(
         )
 
         val container = createContainerForLazyResolve(
-            moduleContext,
-            declarationProviderFactory,
-            CodeAnalyzerInitializer.getInstance(moduleContext.project).createTrace(), // BindingTraceContext(/* allowSliceRewrite = */ true),
-            moduleDescriptor.platform!!,
-            JsPlatformAnalyzerServices,
-            targetEnvironment,
-            languageVersionSettings
+            moduleContext = moduleContext,
+            declarationProviderFactory = declarationProviderFactory,
+            bindingTrace = CodeAnalyzerInitializer.getInstance(moduleContext.project).createTrace(), // BindingTraceContext(/* allowSliceRewrite = */ true),
+            platform = moduleDescriptor.platform!!,
+            analyzerServices = JsPlatformAnalyzerServices,
+            targetEnvironment = targetEnvironment,
+            languageVersionSettings = languageVersionSettings,
+            absentDescriptorHandlerClass = null
```

Comment:
Should the function parameter be passed here? Same thing in `DokkaNativeResolverForModuleFactory`

```suggestion
            absentDescriptorHandlerClass = absentDescriptorHandlerClass
```